source/jormungandr/jormungandr/timezone.py

```diff
@@ -50,7 +51,7 @@ def set_request_instance_timezone(instance):
     if not instance.timezone:
         logger.warn("region {} has no timezone".format(instance.name))
         try:
-            g.timezone = None
+            request.timezone = None
```

Comment:
since request is also a thread local storage does this change anything?

Reply:
you're right, It doesn't change anything, i just found it' would be more meaningful to put it in `request`